azure-iot-device/tests/iothub/test_sync_handler_manager.py

```diff
@@ -164,33 +164,43 @@ def test_handler_invoked_multiple(self, mocker, handler_name, handler_manager, i
         "Is invoked for every item already in the corresponding Inbox at the moment of handler removal"
     )
     def test_handler_resolve_pending_items_before_handler_removal(
-        self, mocker, handler_name, handler_manager, inbox
+        self, mocker, handler_name, handler_manager, handler, inbox
```

Comment:
This change here is unrelated to the rest of the PR - it's a fix for a flaky test that was giving me trouble upon trying to check in